db/src/main/kotlin/edu/artic/db/models/ArticAudioFile.kt

```diff
@@ -36,5 +45,72 @@ data class ArticAudioFile(
             @Json(name = "audio_filesize") val fileSize: String?,
             @Json(name = "audio_transcript") val transcript: String?,
             @Json(name = "credits") val credits: String?
-    ) : Parcelable
-}
\ No newline at end of file
+    ) : Parcelable, BaseTranslation {
+        override fun underlyingLanguage(): String? {
+            return language
+        }
+    }
+
+    /**
+     * Retrieve _all_ of the translations of this content, in one
+     * ordered list.
+     *
+     * Note that this uses the (safe) assumption that [ArticAudioFile]
+     * itself is an English translation of the content.
+     */
+    fun allTranslations() : List<AudioTranslation> {
+        return translations.mapTo(mutableListOf(asAudioTranslation())) {
+            it.asAudioTranslation()
+        }
+    }
+}
+
+
+fun ArticAudioFile.asAudioTranslation() : AudioTranslation {
+    return AudioTranslation(
+            "en-US",
+            title,
+            fileName,
+            fileUrl,
+            fileMime,
+            fileSize,
+            transcript,
+            credits
+    )
+}
+
+fun ArticAudioFile.Translation.asAudioTranslation() : AudioTranslation {
+    return AudioTranslation(
+            language,
```

Comment:
since these are all strings, would it make sense to use named arguments in the construction here? simple intention adds it, just in case the parameters of the `AudioTranslation` change order, we might not get compile errors.